pkg/nodejs/yarn/parse.go

```diff
@@ -60,6 +89,43 @@ func validProtocol(protocol string) (valid bool) {
 	return false
 }
 
+func parseResults(yarnLibs map[string]Library, dependsOn map[string][]Dependency) (libs []types.Library, deps []types.Dependency) {
+	// find dependencies by locators
+	for libLoc, lib := range yarnLibs {
+		libs = append(libs, types.Library{
+			Name:    lib.Name,
+			Version: lib.Version,
+			Locations: []types.Location{
+				lib.Location,
+			},
+		})
```

Comment:
Hello @VaismanLior 
i missed 1 moment: you need to add `ID` here.

Reply:
fixed 👍🏼